orc8r/tools/docker/promote.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -ex
+
+MAGMA_TAG=1.7
+NEW_MAGMA_TAG=1.7.0
```

Comment:
another place where we need to bump versions ;)

Reply:
Version updated to 1.7.1